ArraysHomework/ArraysHomework.cs

```diff
@@ -146,6 +161,7 @@ public void Lists()
             }
 
             // Your code here: combine copy1 into copy 2
+            copy2 = copy1.Concat(copy2).ToList();
```

Comment:
Concat works here.
Have you looked at AddRange()? It should be a little simpler.